app/src/main/java/com/eventyay/organizer/core/attendee/list/SwipeController.java

```diff
@@ -0,0 +1,88 @@
+package com.eventyay.organizer.core.attendee.list;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.RectF;
+import android.support.v7.widget.RecyclerView;
+import android.support.v7.widget.helper.ItemTouchHelper;
+import android.view.View;
+
+import com.eventyay.organizer.R;
+import com.eventyay.organizer.data.attendee.Attendee;
+import com.eventyay.organizer.utils.Utils;
+import com.mikepenz.fastadapter.FastAdapter;
+
+public class SwipeController extends ItemTouchHelper.SimpleCallback {
+
+    private final AttendeesPresenter attendeesPresenter;
+    private final Paint paintGreen = new Paint();
+    private final Paint paintRed = new Paint();
+    private final Bitmap checkinIcon;
+    private final Bitmap checkoutIcon;
+
+    public SwipeController(AttendeesPresenter attendeesPresenter, Context context) {
+        super(0, ItemTouchHelper.RIGHT);
+        this.attendeesPresenter = attendeesPresenter;
+
+        checkinIcon = Utils.drawableToBitmap(context.getResources().getDrawable(R.drawable.ic_checkin));
+        checkoutIcon = Utils.drawableToBitmap(context.getResources().getDrawable(R.drawable.ic_checkout));
```

Comment:
Use `R.drawable.done` and `R.drawable.close`

Reply:
Shouldn't we use the checkin/checkout icons?

Reply:
Changed